bower.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I think `bower` is excess in the project because `npm` solves dependencies issues very well.